docs/data/product/dea-burnt-area-characteristic-layers-sentinel-2-near-real-time/_data.yaml

```diff
@@ -3,20 +3,23 @@
 
 # Overview
 
-title: DEA Burnt Area Characteristic Layers Sentinel-2 Near Real-Time
-long_title: Geoscience Australia Burnt Area Characteristic Layers Sentinel-2 Near Real-Time
+short_name: DEA Burnt Area Characteristic Layers Sentinel-2 Near Real-Time
+full_technical_name: Geoscience Australia Burnt Area Characteristic Layers Sentinel-2 Near Real-Time
 header_image: /_files/cmi/ga_s2_ba_provisional_3_interim_thumbnail.jpg
 version: 1.0.0
 is_latest_version: true
 latest_version_link: null
 is_provisional: true
-product_type: Derivative
-spatial_data_type: Raster
+type:
+  product_type: Derivative
+  spatial_data_type: Raster
 time_span:
   start: 1 Jul 2021
   end: 29 Aug 2023
-update_frequency: Daily
-next_update: No updates planned
+data_updates:
+  frequency: DAILY
+  activity: NO_UPDATES
```

Comment:
@robbibt This product is marked as 'NO_UPDATES', yet it is a Provisional product which intends to be officially released in future. should I instead set mark it as `Paused (updates are temporarily paused)`

Reply:
```suggestion
  activity: PAUSED
```